lib/docker-stdio.js

```diff
@@ -171,7 +174,8 @@ function spawnProcess(opts) {
 
 /**
  * Start up a zlogin'd tar within the user's container then start a TCP server
```

Comment:
The comment would be more clear if it clarified that it's not always a zlogin'd tar, but varies with the zone state.